changelog.md

```diff
@@ -179,6 +179,20 @@ when Marathon fails to clean up reservation which requires offers being sent.
 
 ## Changes from 1.7.xxx to 1.8.180
 
+### Aligning Ephemeral with Stateful Task Handling
+
+Marathon 1.8 introduces handling ephemeral instances similar to how it handled stateful instances since version 1.0. Until now, Marathon expunged instances once all of their tasks ended up in a terminal state, and eventually launched replacements as a result of those instances being expunged. Instances will now only be expunged from the state when their goal is to be Decommissioned and all their tasks are in a terminal state. If their goal is still Running, they will be considered to be scheduled and used to launch replacement tasks. This change not only merges two previously different code paths; this also simplifies debugging since users will be able to follow the task incarnations for a given instance throughout Marathons logs.
```

Comment:
```suggestion
Marathon 1.8 introduces handling ephemeral instances similar to how it handled stateful instances since version 1.0. Until now, Marathon expunged ephemeral instances once all of their tasks ended up in a terminal state, and eventually launched replacements as a result of those instances being expunged. Instances will now only be expunged from the state when their goal is to be Decommissioned and all their tasks are in a terminal state. If their goal is still Running, they will be considered to be scheduled and used to launch replacement tasks. This change not only merges two previously different code paths; this also simplifies debugging since users will be able to follow the task incarnations for a given instance throughout Marathons logs.
```